pyccel/codegen/printing/ccode.py

```diff
@@ -1233,14 +1234,12 @@ def _print_NumpyUfuncBase(self, expr):
             numpy.cos(x) ==> cos(x)
 
         """
-        # add necessary include
+        self.add_import(c_imports['math'])
```

Comment:
Is the math library needed for all `NumpyUfuncBase` functions? For example, is it needed for `sign`?

Reply:
https://github.com/pyccel/pyccel/pull/1169/commits/b4ce59313ab7007e6a6e695b0122457cd329f609 this is not needed anymore as moved it to `_print_NumpySign`